docs/_static/css/sidebar.css

```diff
@@ -0,0 +1,67 @@
+/* sidebar structure */
```

Comment:
one minor thing I noticed: should we put little padding at the bottom of the menu `wy-menu wy-menu-vertical`  ? 
it's little hard to click on the last item if you have a narrow screen (too close to the sticky `version` section)

<img width="301" alt="Screen Shot 2020-02-27 at 9 53 43 PM" src="https://user-images.githubusercontent.com/1120047/75491899-94920300-59ae-11ea-97a5-c9a9ca90a9cd.png">

Reply:
done. meant to add padding for open state of versions... but this would require unnecessary padding while closed.. added minimal padding for now